tests/test_supply_curve_compute.py

```diff
@@ -328,3 +330,154 @@ def test_multi_parallel_trans():
             lcot_1 = sc_1.loc[(sc_1['sc_gid'] == gid), 'lcot'].values[0]
             lcot_2 = sc_2.loc[(sc_2['sc_gid'] == gid), 'lcot'].values[0]
             assert lcot_2 > lcot_1
+
+
+def test_least_cost_full_with_reinforcement():
+    """
+    Test full supply curve sorting with reinforcement costs in the
+    least-cost path transmission tables
+    """
+    with tempfile.TemporaryDirectory() as td:
+        trans_tables = []
+        for cap in [100, 200, 400, 1000]:
+            in_table = os.path.join(TESTDATADIR, 'trans_tables',
+                                    f'costs_RI_{cap}MW.csv')
+            in_table = pd.read_csv(in_table)
+            out_fp = os.path.join(td, f'costs_RI_{cap}MW.csv')
+            in_table["reinforcement_cost"] = 0
+            in_table["reinforcement_dist_km"] = 0
+            in_table.to_csv(out_fp, index=False)
+            trans_tables.append(out_fp)
+
+        sc_full = SupplyCurve.full(SC_POINTS, trans_tables, fcr=0.1,
+                                   avail_cap_frac=0.1,
+                                   columns=('trans_gid', 'trans_capacity',
+                                            'trans_type',
+                                            'trans_cap_cost_per_mw',
+                                            'dist_km', 'lcot', 'total_lcoe',
+                                            'max_cap', 'n_parallel_trans'))
+
+        fpath_baseline = os.path.join(TESTDATADIR, 'sc_out/sc_full_lc.csv')
+        baseline_verify(sc_full, fpath_baseline)
+        verify_trans_cap(sc_full, trans_tables)
+
+        trans_tables = []
+        for cap in [100, 200, 400, 1000]:
+            in_table = os.path.join(TESTDATADIR, 'trans_tables',
+                                    f'costs_RI_{cap}MW.csv')
+            in_table = pd.read_csv(in_table)
+            out_fp = os.path.join(td, f'costs_RI_{cap}MW.csv')
+            in_table["reinforcement_cost"] = 1e6
+            in_table["reinforcement_dist_km"] = 10
```

Comment:
really like how weve been doing this lately - add tests that manipulate the existing files with simple checks instead of adding new baseline inputs/outputs

Reply:
Me too :) And the code spells out what is important about the data in the file, so you don't have to go digging thought the file manually to find out